backend/config/corpora-api.yml

```diff
@@ -486,10 +491,16 @@ paths:
               schema:
                 type: object
                 properties:
-                  dataset_uuid:
+                  dataset_id:
```

Comment:
I've been seeing uuid and id used interchangeably in the spec. In some areas we are using `id`, but when referring to a `id` with other `id`s in the context we've been using `{name}_uuid`. Since there are no other `id`s in the context of this response I justified to a myself a hybrid of `{name}_id` which follows no convention to so far. I think we should decide whether to use `uuid` or `id` in the spec, not both. I think `id` is sufficient, but would require more refectoring since `uuid` is more widely used in the code already. Changing to either one would cause some thrash with the frontend and backend since the API is already being used. For now I'll change it back to `dataset_uuid`